### PR TITLE
[doc]: Correct the form of `contract-in`

### PR DESCRIPTION
**Pull request by NoahStoryM**
##### Checklist

- [ ] Bugfix
- [ ] Feature
- [ ] tests included
- [x] documentation

### Description of change

`(contract-in in-out-item ...)` -> `(contract-in module-path in-out-item ...)`

### CODE REVIEW

File: pkgs/racket-doc/scribblings/reference/contracts.scrbl [hunk 1]
```diff
@@ -1944,8 +1944,8 @@ earlier fields.}}
 
 @deftogether[(@defform[
  #:literals (struct rename)
- (contract-in in-out-item ...)]
-               @defform[
+ (contract-in require-spec in-out-item ...)]
```
Comment (sorawee) [inline pkgs/racket-doc/scribblings/reference/contracts.scrbl:1947]: I don't think `require-spec` is right. It must be a module path. We can't do:

```
(require (contract-in (only-in racket add1)
                      [add1 (-> number? number?)]))
```

which fails with:

```
contract violation
  expected: (and/c syntax? (lambda (s) (module-path? (syntax->datum s))))
  given: #<syntax:3-unsaved-editor:3:22 (only-in racket add1)>
```

**Comment by sorawee** [comment]
Thanks!

**Comment by sorawee** [comment]
Ughh, I should have squashed the commits. My bad :/

**Comment by rfindler** [comment]
Thanks!

**Pull request merged**